docs/Developers/Git_Basics.rst

```diff
@@ -70,7 +70,10 @@ Creating a new branch
 ---------------------
 
 To start working on an issue, you first need to create a new branch where you
```

Comment:
Maybe someone completely new to git would confuse the first 2 sentences as the newly created branch as master.